FAQ.md

```diff
@@ -12,29 +12,29 @@ Frequently Asked Questions
 - [What to do in case of errors?](#what-to-do-in-case-of-errors)
 - [Where do I seek support?](#where-do-i-seek-support)
 
-## What are the relations with OCP project? Is OKD4 being upstream for OCP?
+## What are the relations with OCP project? Is OKD4 an upstream of OCP?
 
 In 3.x release time OKD was used as an upstream project for Openshift Container Platform. OKD could be installed on 
-Fedora/CentOS/RHEL and used CentOS based images to install the cluster. However OCP could be installed on RHEL only and it rebuilt all the images so that these would be RHEL-based.
+Fedora/CentOS/RHEL and used CentOS based images to install the cluster. OCP, however, could be installed only on RHEL and its images were rebuilt to be RHEL-based.
 
-[Universal Base Image project](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) has enabled us to run RHEL-based images on any platform, so the full image rebuilt is no longer necessary. This allows OKD4 project to reuse most images from OCP4. However OCP also has another critical part - Red Hat RHEL CoreOS. Although its an opensource project (since its essentially RHEL8) its not a community-driven project. As a result OKD workgroup has 
-made a decision to use Fedora CoreOS - opensource and community-driven project - as a base for OKD4. This decisions allows end-user modify all parts of the cluster using prepared instructions.
+[Universal Base Image project](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image) has enabled us to run RHEL-based images on any platform, so the full image rebuild is no longer necessary, allowing OKD4 project to reuse most images from OCP4. There is another critical part of OCP - Red Hat Enterprise Linux CoreOS. Although it's an open source project (since it's essentially RHEL8) it's not a community-driven project. As a result OKD workgroup has 
```

Comment:
>Although it's an open source project (since it's essentially RHEL8) it's

While we are here lets rephrase it - `.. its ... its ... its ...`?

Reply:
reworked.